eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/core/protocol/tcp/client/group/ClientGroupWrapper.java

```diff
@@ -249,7 +227,7 @@ public boolean removeSubscription(SubscriptionItem subscriptionItem, Session ses
         String topic = subscriptionItem.getTopic();
         if (session == null
             || !StringUtils.equalsIgnoreCase(group,
-                EventMeshUtil.buildClientGroup(session.getClient().getGroup()))) {
+            EventMeshUtil.buildClientGroup(session.getClient().getGroup()))) {
```

Comment:
```suggestion
if (session == null || !StringUtils.equalsIgnoreCase(group, EventMeshUtil.buildClientGroup(session.getClient().getGroup()))) {
```

The same as L333-L335, L365-L367, L395-L397, L427-L429.